### ISSUE TITLE
[Bug]: experimental-addon-test runner panel issues

### ISSUE THREAD

**Issue by SteveALee** [storybookjs/storybook#30536]
### Describe the bug

![Image](https://github.com/user-attachments/assets/74632126-aca0-47ff-b7ac-24ea5fa0596f)

- what are 'local' tests? None are remote are they? I'm not sure what these vitest tests are called compare to the iteractive host browser tests
- when all test pass the resuts are shown as a green fill to the list marker. This is accessibility failure [WCAG SC 1.2.1](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color) as not run and pass only differ by colour. If test fail a count of failures does appear but at a distance away 

![Image](https://github.com/user-attachments/assets/a8fd910e-e319-4b71-92a0-467b5d78923e)

 

### Reproduction link

N/A

### Reproduction steps

Run any tests and expand the "Run Tests" panel

### System

```bash
System:
    OS: Windows 11 10.0.26100
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
  Binaries:
    Node: 22.12.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 10.9.0 - C:\Program Files\nodejs\npm.CMD <----- active
    pnpm: 10.0.0 - ~\AppData\Local\pnpm\pnpm.EXE
  Browsers:
    Edge: Chromium (127.0.2651.74)
  npmPackages:
    @storybook/addon-a11y: ^8.5.5 => 8.5.5
    @storybook/addon-essentials: ^8.5.5 => 8.5.5
    @storybook/addon-svelte-csf: ^5.0.0-next.23 => 5.0.0-next.23
    @storybook/blocks: ^8.5.5 => 8.5.5
    @storybook/experimental-addon-test: ^8.5.5 => 8.5.5
    @storybook/svelte: ^8.5.5 => 8.5.5
    @storybook/svelte-vite: ^8.5.5 => 8.5.5
    @storybook/test: ^8.5.5 => 8.5.5
    storybook: ^8.5.5 => 8.5.5
```

**Comment by Juanescacha** [comment]
I also have been struggling to find out how to by default enable "Accessability" and "Coverage" tests for this Run Panel, haven't found anything?

**Comment by grissinibread** [comment]
I'm working on this

**Comment by shilman** [comment]
We're planning to solve this in a different way in the coming release.

1. The checkboxes to toggle the setting will always be visible, so it will be easier to enable a11y
2. When you enable a11y tests, your setting will be persisted across Storybook sessions

We don't want to expand the product surface area with yet another setting.

**Comment by grissinibread** [comment]
We forked the repository and followed the **new contribution guide** that was linked at the very bottom of the **original contribution guide** in the repository using **Github Codespaces**. 


However, we’ve run into a couple of issues. Firstly, we couldn’t get ‘yarn start’ to work. We had to go into the main package.json file in order to manually change the yarn version of 1.22.22 to 4.6.0. That seemed to have fixed the issue.

**Issue:**
However, after running the command  ‘yarn test’, to make sure everything is working as it should before making any new code changes, we get many errors. This is what we initially get before the terminal outputs seemingly the same error over and over again, which makes us unable to view this text even when scrolling up.

<img width="1212" alt="Image" src="https://github.com/user-attachments/assets/6aa9a1ca-f4fb-4b0d-b771-e6f269943831" />

While we can’t put the entire output on here, as its too large, here is a screenshot of the output of one of the cases:

<img width="1209" alt="Image" src="https://github.com/user-attachments/assets/2cafb3e4-ede6-4071-8316-2867d709e23f" />

This is obviously a problem as the contributing guide specifically states “Before jumping onto any code changes, verifying everything is working is essential—specifically, the test suite.”

**What we’ve tried so far:**
We looked through the documentations, but have found nothing that has benefited us in solving this issue so far. We have also attempted to update the ‘STORYBOOK_URL’ inside ‘code/.env’ file to the port we’ve been using to run the application when running these tests, but that doesn’t seem to be the issue either. Once again we are attempting to run this on **Github CodeSpaces**, so we’re not sure if we’re missing something or if the project isn’t supported on the platform. Any and all help would be greatly appreciated. Thank you!

**Comment by shilman** [comment]
@grissinibread Did you have corepack enabled?

**Comment by grissinibread** [comment]
Yes @shilman, I made sure to enable corepack

**Comment by gian-benito** [comment]
@shilman Has this issue been solved in the latest release? The accessibility and coverage tests being disabled have been fixed by the added checkboxes. I believe accessibility is no longer an issue because you can hover over each status icon and it will direct you to its test details.

**Comment by shilman** [comment]
@gian-benito thanks for the ping! Yes the new UI is available in the 9.0-alpha release. It's a lot more usable. I have not verified that it solves all accessibility problems, but I'm closing this issue. If there accessibility problems with the new UI, please file a new issue detailing them!
